bin/post_compile

```diff
@@ -9,4 +9,12 @@ if [ -f bin/run_collectstatic ]; then
     bin/run_collectstatic
 fi
 
+echo "-----> Running manage.py check --deploy --fail-level WARNING"
+python manage.py check --deploy --fail-level WARNING
+
+if [ -n "$AUTO_MIGRATE" ]; then
+    echo "-----> Running manage.py migrate"
+    python manage.py migrate
```

Comment:
Best to use `python manage.py migrate --noinput` here, it'll make it easier when doing deploys of applications that are using our boilerplate. Basically, it'll avoid asking the user for inputs when running post_compile, which is the desirable thing to do here.